### DISCUSSION TITLE
Visual overhaul, alignment and individual line translation

### DISCUSSION THREAD

**Discussion by machinewrapped** [machinewrapped/llm-subtrans#119] [Announcements]
This version contains a substantial visual overhaul. Original and translated text are now consolidated into a unified widget so that they are always aligned even when only part of the text is translated. Each line now shows the length of the subtitle and the length of the gap before it.

A footer has been added to each scene and batch showing how many lines have been translated. Additionally, individual lines can now be (re)translated by  selecting them and using the "Translate Selection" button.

Spacing and margins have been tweaked for visual cohesion and better use of space. Dark mode has new colours.

<hr /><em>This discussion was created from the release <a href='https://github.com/machinewrapped/gpt-subtrans/releases/tag/v0.5.8'>Visual overhaul, alignment and individual line translation</a>.</em>

**Comment by Neoony** [comment]
Working good and looking nice on v0.5.8 (the visual overhaul)
Well done :)
